source/clear-linux/tutorials/hadoop.rst

```diff
@@ -0,0 +1,177 @@
+.. _hadoop:
+
+Set up a single node cluster with Hadoop\*
+##########################################
+
+This tutorial walks you through the process of installing, configuring, and
+running Apache\* Hadoop on |CLOSIA|. The Apache Hadoop software library is a
+framework for distributed processing of large data sets across clusters of
+computers using simple programming models. It is designed to scale up from
+single servers to thousands of machines, with each machine offering local
+computation and storage.
+
+Prerequisites
+*************
+
+Before following this tutorial, you should follow the
+:ref:`bare-metal-install` to ensure you have installed |CLOSIA|.
+
+Before you install any new packages, update |CL| with the following command:
+
+.. code-block:: bash
+
+   sudo swupd update
+
+For the purposes of this tutorial, we will install Hadoop in a single machine
+running both the master and slave daemons.
+
+Install Apache Hadoop
+*********************
+
+Apache Hadoop is included in the `big-data-basic` bundle. To install the
+framework, enter the following command:
+
+.. code-block:: bash
+
+   sudo swupd bundle-add big-data-basic
+
+Configure Apache Hadoop
+***********************
+
+#. To create the configuration directory, enter the following command:
+
+   .. code-block:: bash
+
+      sudo mkdir /etc/hadoop
+
+#. Copy the defaults from :file:`/usr/share/defaults/hadoop` to
+   :file:`/etc/hadoop` with the following command:
+
+   .. code-block:: bash
+
+      $ sudo cp /usr/share/defaults/hadoop/* /etc/hadoop
+
+   .. note:: Remember, you should never modify the files under the
```

Comment:
I would mention the reason why is because we are a stateless OS in this note.  I try to mention that whenever it could potentially be an issue for a user

Reply:
Done